ui/components/component-library/banner-tip/README.mdx

```diff
@@ -0,0 +1,146 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { BannerTip } from './banner-tip';
+import { BannerBase } from '..';
+
+# BannerTip
+
+`BannerTip` is an inline notification that offers users educational tips, knowledge, and helpful links
+
+<Canvas>
+  <Story id="components-componentlibrary-bannertip--default-story" />
+</Canvas>
+
+## Props
+
+The `BannerTip` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
+
+<ArgsTable of={BannerTip} />
+
+The `BannerTip` accepts all `BannerBase` component props below
+
+<ArgsTable of={BannerBase} />
+
+### Logo Type
+
+Use the `logoType` prop with the `BannerTipLogoType` object from `../../component-library` to change the context of `BannerTip`.
+
+Possible options:
+
+- `BannerTipLogoType.greeting` Default
+- `BannerTipLogoType.chat`
+
+<Canvas>
+  <Story id="components-componentlibrary-bannertip--logo-type" />
+</Canvas>
+
+```jsx
+import { BannerTip } from '../../component-library';
+
+<BannerTip logoType={BannerTipLogoType.greeting}>
+  This is a demo of chat.
+</BannerTip>
+<BannerTip logoType={BannerTipLogoType.chat}>
+  This is a demo of chat.
+</BannerTip>
+```
+
+### Title
+
+Use the `title` prop to pass a string that is sentence case no period. Use the `titleProps` prop to pass additional props to the `Text` component.
+
+<Canvas>
+  <Story id="components-componentlibrary-bannertip--title" />
+</Canvas>
+
+```jsx
+import { BannerTip } from '../../component-library';
+
+<BannerTip title="Title is sentence case no period">
+  Pass only a string through the title prop
+</BannerTip>;
```

Comment:
Also happens to me as well. It will be something to do with eslint rule I think it's low priority for now. I'll create an issue to investigate if we can update the settings/